docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,495 @@
+In addition to your knowledge about GitHub and creating a pull request, we have
+specific conventions and requirements you need to follow when submitting a pull
+request for LoopBack 4.
+
+#### 1. Commits to local repository
+
+Our commit messages are formatted according to
+[Conventional Commits](https://conventionalcommits.org/).
+
+Please read the
+[Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines)
+guidelines to correctly format your commit messages.
+
+To change an existing commit please refer to
+[Changing a commit message](https://help.github.com/articles/changing-a-commit-message/).
+
+#### 2. Before pushing to remote repository
+
+Before pushing to the remote repository, ensure your files are free of
+formatting, syntax, and logical/execution errors, by running :
+
+```
+npm run lint:fix && npm run test
+```
+
+#### 3. After creating PR, sign the CLA and fill out checklist
+
+After creating the pull request, sign the Contributor License Agreement (CLA).
+
+Please read
+[Agreeing to the CLA](https://loopback.io/doc/en/contrib/code-contrib.html#agreeing-to-the-cla),
+click on this
+[link](https://cla.strongloop.com/agreements/strongloop/loopback-next), and sign
+it. This is done once per repository.
+
+The pull request has a checklist that must be filled out appropriately.
+
+![submit_pr_template_1.png](./imgs/submit_pr_template_1.png)
+
+Of particular importance are the `'npm test passes on your machine'` and
+`'Sign the CLA (Contributor License Agreement)'` checkboxes.
+
+#### 4. Check CI status
+
+Ensure that the continuous integration (CI) jobs associated with your pull
+request complete successfully.
+
+![submit_pr_final_commit_ci_jobs_pass_1.png](./imgs/submit_pr_final_commit_ci_jobs_pass_1.png)
+
+#### 5. PR Review Process
+
+The review process is iterative, so it is normal that `multiple commits` may be
+required until the pull request is finally accepted.
+
+{% include tip.html content="After a pull request is created, any additional commits that you push to your remote feature branch automatically get added to the pull request. There is no need to close the initial pull request and create a new one." %}
+
+Reactively [rebase](https://help.github.com/articles/about-git-rebase/) your
+forked repository against the upstream master branch to keep them in synch; if
+needed.
+
+#### Before your PR is merged by maintainer
+
+Before the PR is merged by a
+maintainer,[squash your commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)
+into one commit or a few commits; if needed. This step helps to keep the
+repository's commit history clean and concise.
+
+### Beginner Instructions
+
+Here is a complete tutorial on how to submit a pull request (PR) for LoopBack
+v4.
+
+#### 1. Fork the loopback-next repository
+
+In your browser, navigate to : <https://github.com/strongloop/loopback-next> .
+
+Create your own fork of the repository, by pressing the `Fork` link on the
```

Comment:
nitpick: I'd leave out the comma after repository and also remove space between `side` and `.`